papertrail_cmd.go

```diff
@@ -1,19 +1,32 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"github.com/RackSec/srslog"
+	"github.com/buger/jsonparser"
+	"github.com/urfave/cli"
 	"log"
 	"strconv"
 	"strings"
 	"time"
-
-	"github.com/RackSec/srslog"
-	"github.com/buger/jsonparser"
-	"github.com/urfave/cli"
 )
 
 var papertrailprefix string
 
+type PapertrailLog struct {
+	Message       string  `json:"message"`
+	Component     string  `json:"component"`
+	Severity      string  `json:"severity"`
+	Action        string  `json:"action"`
+	Subject       string  `json:"subject"`
+	CorrelationId string  `json:"correlation_id"`
+	MessageId     string  `json:"message_id"`
+	Stream        string  `json:"stream"`
+	Time          int64   `json:"time"`
```

Comment:
in the node implementation we actually use new Date().toISOString(); for time.

Reply:
Seems whatever is adding the kube stuff is overriding the field.